content/docs/guides/data-package-csvw.md

```diff
@@ -0,0 +1,191 @@
+---
+title: How does Data Package compare with CSVW
+sidebar:
+  order: 3
+---
+
+<table>
+  <tr>
+    <th>Authors</th>
+    <td>Peter Desmet, Evgeny Karev, Sara Petti</td>
+  </tr>
+</table>
+
+In 2016, a working group chartered by W3C created a set of recommendations for documenting and accessing CSV files, collectively called [CSVW on the Web (CSVW)](https://www.w3.org/TR/2016/NOTE-tabular-data-primer-20160225/). These show extensive similarities with the Data Package standard. Below we give an overview of the (dis)similarities between the two.
+
+## Scope
+
+Data Package is a set of specifications to describe datasets, data files and tabular data. It can describe data files that are not tabular and includes extra specifications for those that are (i.e. [Table Dialect](/standard/table-dialect/) and [Table Schema](/standard/table-schema/)).
+
+CSVW is a set of specifications to describe delimited text files (CSV, TSV) only.
```

Comment:
, in particular the [Model for Tabular Data and Metadata on the Web](https://www.w3.org/TR/tabular-data-model/) and the [Metadata Vocabulary for Tabular Data](https://www.w3.org/TR/tabular-metadata/).

Reply:
Thanks, will refer to the versioned URLS + mention it also has two transformation specs (to JSON and to RDF).